cardano-testnet/test/cardano-testnet-test/files/plutus/v3/always-succeeds.plutus

```diff
@@ -1,5 +1,5 @@
 {
     "type": "PlutusScriptV3",
-    "description": "",
-    "cborHex": "484701010022280001"
-}
+    "description": "Always succeeds",
+    "cborHex": "46450101002499"
```

Comment:
what has changed in the script?

Reply:
I don't know. I just created the new one from scratch, and the one that was there was not working, it wasn't returning the right thing (`BuiltinUnit`), apparently, and this one does work 🤷

Reply:
Also I don't know where the old one came from. We could try to inspect it as Plutus Core, but I couldn't be bothered